src/key.rs

```diff
@@ -1016,6 +1016,8 @@ impl Keypair {
 
     /// Constructs a schnorr signature without aux rand for `msg` using the global
     /// [`SECP256K1`] context.
+    /// ⚠️ **Warning:** Signing multiple messages with the same private key without a unique nonce
+    /// can expose the private key. **This function signs the message without a random nonce!**
```

Comment:
nit: Needs a line of whitespace between the heading and body.

```suggestion
    ///
    /// ⚠️ **Warning:** Signing multiple messages with the same private key without a unique nonce
    /// can expose the private key. **This function signs the message without a random nonce!**
```

I'll leave review of the content to Andrew.